META-INF/plugin.xml

```diff
@@ -635,6 +635,10 @@ We keep delivering funded results =)
         shortName="UsageOfSilenceOperatorInspection"              displayName="Usage of the silence operator"
         groupName="Code Style"                                    enabledByDefault="true" level="WARNING"
         implementationClass="com.kalessil.phpStorm.phpInspectionsEA.inspectors.codeSmell.UsageOfSilenceOperatorInspector"/>
+    <localInspection language="PHP" groupPath="PHP,Php Inspections (EA Extended)"
+        shortName="MagicNumberInspection"                         displayName="Magic number usage"
+        groupName="Code Style"                                    enabledByDefault="true" level="WEAK WARNING"
```

Comment:
All of them are magic numbers (it mean time-related values). Then if you are having problems like that, you should refactoring your code to centralize this kind of value in an unique class. For instance:

```php
class TimeConstants {
    public const HOURS_BY_DAY = 24;
    public const MINUTES_BY_HOUR = 60;
    public const SECONDS_BY_MINUTE = 60;
    // So on...
}
```

And I do not understand your last question (same for your last comment ("I just started the review...")). You mean that you will try another PR before this? If do, no problem, do that.